Termin 03/frontend/src/features/users/hooks/useUserCreatedToast.ts

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from "react";
+import { io, Socket } from "socket.io-client";
+import { toast } from "react-toastify";
+import type { User } from "../types";
+
+const SOCKET_URL = import.meta.env.VITE_API_BASE_URL || "http://localhost:5000";
```

Comment:
Postoji problem u ovoj liniji koda. Trenutno se čita putanja `VITE_API_BASE_URL=http://localhost:5000/api/v1` a za WebSocket treba da bude `http://localhost:5000`. Treba da dodaš novu `.env` varijablu npr. `VITE_SOCKET_URL`. **Obrati pažnju** da je moraš dodati i u `.env.example` kako bi druge kolege koje kloniraju repozitorijum znale kako da postave njihov`.env` fajl.

```suggestion
const SOCKET_URL = import.meta.env.VITE_SOCKET_URL || "http://localhost:5000";
```